vsintegration/src/FSharp.Editor/Common/LanguageService.fs

```diff
@@ -48,7 +48,7 @@ type internal FSharpCheckerProvider
     ) =
     let checker = 
         lazy
-            let checker = FSharpChecker.Create()
+            let checker = FSharpChecker.Create(projectCacheSize = 200, keepAllBackgroundResolutions = false)
```

Comment:
@smoothdeveloper No, that setting is the one used if none is provided explicitly at FSharpChecker creation.   So this changes the setting from 3 to 200 (which means basically strongly holding all background checkers and associated resources for all projects - which intuitively feels sure to blow up memory usage with many scripts or projects - in F# each script counts as a project)